findGame/TipOnLoadingPage.js

```diff
@@ -7,40 +7,47 @@
 $(document).ready(function() {
     var num = 0;
     var tips = [
-        "When chasing the opponent's flag carrier always stay between them and the flag tile.",
-        "Nobody on regrab? Get on regrab. Somebody on regrab? Chase the player holding your flag.",
+        "When chasing the opponent's flag carrier, try to stay between them and the flag tile.",
+        "Nobody on regrab? Get on regrab. Somebody on regrab? Chase the player holding your flag!",
         "Pressing your keys harder doesn't make you go faster. Don't break your keyboard!",
-        "When you have the flag, it's not always best to cap quickly. Be patient and wait for your team to put in blocks.",
-        "Always have two people stay in base to prevent the other team from taking your flag.",
-        "Stay near the opponent's base so you can regrab the flag if your teammate pops.",
-        "Watch your speed when chasing. Move too fast and you might overcommit. Too slow and they'll go right past you!",
-        "Try to not grab the flag when the opponents are coming in for a capture. If you die there will be one less person to try and stop them.",
-        "Consider waiting if there aren't 8 players in the game to make things fair.",
-        "Powerups respawn one minute after they are last picked up.",
-        "When you have a tagpro on offense let a teammate pick up the flag and focus on clearing a path.",
-        "A rolling bomb powerup works as a one-time shield when you're carrying the flag.",
-        "Rolling on tiles of your color make you move a lot faster.",
-        "Boosts respawn 10 seconds after they are used. Bombs respawn after 30 seconds. Powerups after 1 minute.",
-        "You can press escape to show the scoreboard.",
-        "You can go to the <a href='/groups'>groups</a> page to team up with some friends and play TagPro together.",
-        "Rank points contribute to your score on the <a href='/boards'>leaderboard</a>.",
-        "You can set a name and choose a flair on the profile page.",
-        "If your reserved name is the same as your displayed name, your name will turn green and have a check mark next to it.",
+        "When you have the flag, timing is important. Be patient and wait for your team to set up blocks.",
+        "Have two people stay on defense to prevent the other team from taking your flag.",
+        "Watch your speed when chasing. If you move too fast you might overcommit. Too slow and they'll go right past you!",
+        "Try to not grab the flag when the opponents are coming in to cap. If you die there's one less person to stop them!",
+        "For a fair game, consider waiting if there are fewer than 6 players.",
```

Comment:
Let's keep it at 8. People complain about 3v4 all the time, don't want to officially suggesting capping then.